### ISSUE TITLE
require is called even when not defined

### ISSUE THREAD

**Issue by haruska** [elliotf/chai-fuzzy#11]
At the top of the index.js (line 3), chai-fuzzy checks for the presence of "require" as a function. If it is not defined, it calls it anyway (line 32).

**Comment by elliotf** [comment]
You're totally right.  Is there a good example of boilerplate for a module with dependencies supporting multiple module loading frameworks, @haruska ?

**Comment by haruska** [comment]
I'm not sure but the line 32 call should be wrapped up in the line 3 call or it should just assume it exists and note in the documentation that it's absolutely required to function.
